lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -98,25 +98,6 @@ sub find_by_subscribed_editor
     $self->query_to_list_limited($query, [$editor_id], $limit, $offset);
 }
 
-sub find_by_area {
-    my ($self, $area_id, $limit, $offset) = @_;
-    my (
-        $containment_query,
-        @containment_query_args,
-    ) = get_area_containment_query('$2', 'any(array[area, begin_area, end_area])', check_all_levels => 1);
-    my $query = 'SELECT ' . $self->_columns . '
-                 FROM ' . $self->_table . "
-                 WHERE \$1 IN (area, begin_area, end_area) OR EXISTS (
-                    SELECT 1 FROM ($containment_query) ac
-                     WHERE ac.descendant IN (area, begin_area, end_area) AND ac.parent = \$1
-                 )
-                 ORDER BY artist.name COLLATE musicbrainz, artist.id";
```

Comment:
I'd consider using:

```
sub _find_by_area_order {
    return 'sort_name, id';
}
```
here while you're at it, but not a big deal either way I guess :)